samples/nrf_desktop/configuration/nrf52840_pca10059/flash.overlay

```diff
@@ -1,36 +0,0 @@
-/* Configuration without nRF5 bootloader. */
```

Comment:
I am not sure this will do. Previously we had problems as default dts for dongle has flash map defined for Nordic bootloader which we don't want to have. Our overlay ensures that we stay with only one bootloader mcuboot and it is located at the beginning of the flash.

Reply:
This patch:

https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/683/files#diff-effed09cf5e7d5741a93f909defb72baL160

enables partition manager. Which again causes partition information to be taken from partition manager, instead of dts. So default dts files can no longer cause problems for nrf_desktop.

Reply:
I build the project on the branch. I used empty flash.overlay after build error. Merge hex indeed shows that bootloader is positioned at 0x0000.

Reply:
To confirm, this is desired behaviour, right?

Reply:
Yes, please ignore this part